clang-tools-extra/docs/clang-tidy/checks/bugprone/unused-local-non-trivial-variable.rst

```diff
@@ -0,0 +1,29 @@
+.. title:: clang-tidy - bugprone-unused-local-non-trivial-variable
+
+bugprone-unused-local-non-trivial-variable
+==========================================
+
+Warns when a local non trivial variable is unused within a function.
+
+In the following example, `future2` would generate a warning that it is unused.
+
+.. code-block:: c++
+
+   std::future<MyObject> future1;
+   std::future<MyObject> future2;
+   // ...
+   MyObject foo = future1.get();
+   // future2 is not used.
+
+Options
+-------
+
+.. option:: IncludeTypeRegex
+
+   Semicolon-separated list of regular expressions matching types of variables to check.
+   By default it 'std::.*mutex;std::future'.
```

Comment:
Please use single back-ticks for option names and values.

Reply:
Done.